modules/contentbox/migrations/2022_10_14_173737_CONTENTBOX-1429-Slow-Admin-Search.cfc

```diff
@@ -0,0 +1,12 @@
+component {
+
+	function up( schema, qb ){
+		schema.alter( "cb_contentVersion", function( table ){
+			table.index( [ "isActive", "content" ], "idx_publishedContentSearch" );
```

Comment:
dumb question: what happens if the index already exists? Will the db ignore it?

Reply:
No, it will throw an error. Since this index is named here in this migration, though, the only way the index would exist is for this file to execute.